### PR TITLE
chore: add a doc about how to generate doc PDF locally 

### PR DESCRIPTION
**Pull request by qiancai**
### What is changed, added or deleted? (Required)

As in title

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: resources/tidb-pdf-generation-tutorial.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
+---
+title: 用户自助生成 TiDB 文档 PDF 教程
+summary: 介绍如何在本地定制输出符合特定场景用户需求的 PDF。
+---
+
```
Comment (qiancai) [inline resources/tidb-pdf-generation-tutorial.md:5]:
```suggestion
```

File: resources/tidb-pdf-generation-tutorial.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+### 准备 2: 将 TiDB 文档仓库克隆到本地
+
+> 大约耗时：10 分钟
+
+TiDB 中文文档仓库的地址为 <https://github.com/pingcap/docs-cn>，英文文档仓库的地址为 <https://github.com/pingcap/docs>。
+
+下面的步骤以克隆 TiDB 中文文档为例：
+
+1. 打开 TiDB 中文文档仓库地址：<https://github.com/pingcap/docs-cn>
+2. 点击页面右上角的 [**Fork**](https://help.github.com/articles/github-glossary/#fork) 按钮，等待 Fork 完成即可。
```
Comment (shawn0915) [inline resources/tidb-pdf-generation-tutorial.md:41]: 这里可以考虑直接放置fork docs-cn的link么？
https://github.com/pingcap/docs-cn/fork

Comment (qiancai) [inline resources/tidb-pdf-generation-tutorial.md:41]:
```suggestion
2. 点击页面右上角的 [**Fork**](https://github.com/pingcap/docs-cn/fork) 按钮，等待 Fork 完成即可。
```

File: resources/tidb-pdf-generation-tutorial.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
+---
+title: 用户自助生成 TiDB 文档 PDF 教程
```
Comment (lilin90) [inline resources/tidb-pdf-generation-tutorial.md:2]:
```suggestion
title: 自助生成 TiDB 文档 PDF 教程
```

File: resources/tidb-pdf-generation-tutorial.md [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+summary: 介绍如何在本地定制输出符合特定场景用户需求的 PDF。
```
Comment (lilin90) [inline resources/tidb-pdf-generation-tutorial.md:3]:
```suggestion
summary: 介绍如何在本地定制输出符合特定场景需求的 TiDB 文档 PDF。
```

File: resources/tidb-pdf-generation-tutorial.md [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+---
+
+# 用户自助生成 TiDB 文档 PDF 教程
```
Comment (lilin90) [inline resources/tidb-pdf-generation-tutorial.md:6]:
```suggestion
# 自助生成 TiDB 文档 PDF 教程
```

File: resources/tidb-pdf-generation-tutorial.md [hunk 6]
```diff
@@ -0,0 +1,109 @@
...
+
+本教程为你提供了一种可以自助生成 TiDB PDF 文档的方法。通过该方法，你可以在本地对 TiDB 文档目录进行自由排序和删减，定制输出符合特定场景用户需求的 PDF。
```
Comment (lilin90) [inline resources/tidb-pdf-generation-tutorial.md:8]:
```suggestion
本教程提供了一种自助生成 PDF 格式的 TiDB 文档的方法。通过该方法，你可以在本地对 TiDB 文档目录进行自由排序和删减，定制输出符合特定场景需求的 PDF。
```

File: resources/tidb-pdf-generation-tutorial.md [hunk 7]
```diff
@@ -0,0 +1,109 @@
...
+
+## 环境准备
+
+这些准备工作只需要在第一次生成 PDF 时进行，以后再次生成 PDF 时可直接跳过。
```
Comment (lilin90) [inline resources/tidb-pdf-generation-tutorial.md:12]:
```suggestion
以下准备工作只需要在第一次生成 PDF 时进行，以后再次生成 PDF 时可直接跳过。
```

File: resources/tidb-pdf-generation-tutorial.md [hunk 8]
```diff
@@ -0,0 +1,109 @@
...
+
+### 准备 1: 安装并配置 Docker 环境
```
Comment (lilin90) [inline resources/tidb-pdf-generation-tutorial.md:14]:
```suggestion
### 准备 1：安装并配置 Docker 环境
```

File: resources/tidb-pdf-generation-tutorial.md [hunk 9]
```diff
@@ -0,0 +1,109 @@
...
+
+> 大约耗时：30 分钟
+
+1. 安装 [Docker](https://docs.docker.com/get-docker/)。
+2. 在 Mac Terminal 或者 Windows powershell 运行 `docker --version` 命令。
```
Comment (lilin90) [inline resources/tidb-pdf-generation-tutorial.md:19]:
```suggestion
2. 在 macOS Terminal 或者 Windows PowerShell 运行 `docker --version` 命令。
```

File: resources/tidb-pdf-generation-tutorial.md [hunk 10]
```diff
@@ -0,0 +1,109 @@
...
+
+    如果看到 Docker 的版本信息，说明安装成功。
+
+3. 配置 Docker 资源：
+    1. 打开 Docker 应用程序，点击右上角的齿轮图标。
+    2. 点击 **Resources**，然后将 **Memory** 设置为 `8.00 GB`。
+4. 在 Mac Terminal 或者 Windows powershell 运行以下命令拉取文档构建的 Docker 镜像：
```
Comment (lilin90) [inline resources/tidb-pdf-generation-tutorial.md:26]:
```suggestion
4. 在 macOS Terminal 或者 Windows PowerShell 运行以下命令拉取文档构建的 Docker 镜像：
```

**Review by lilin90 — COMMENTED** [review]
How about adding this tutorial to `README.md` as a section? 🤔

**Review by lilin90 — APPROVED** [review]
/lgtm

File: resources/tidb-pdf-generation-tutorial.md [hunk 11]
```diff
@@ -0,0 +1,118 @@
...
+2. 在 macOS Terminal 或者 Windows PowerShell 运行 `docker --version` 命令。
+
+    如果看到 Docker 的版本信息，说明安装成功。
+
+3. 配置 Docker 资源。
+
+    1. 打开 Docker 应用程序，点击右上角的齿轮图标。
+    2. 点击 **Resources**，然后将 **Memory** 设置为 `8.00 GB`。
+
+4. 在 macOS Terminal 或者 Windows PowerShell 运行以下命令拉取文档构建的 Docker 镜像：
```
Comment (Oreoxmt) [inline resources/tidb-pdf-generation-tutorial.md:29]:
```suggestion
4. 在 macOS Terminal 或者 Windows PowerShell 运行以下命令拉取 TiDB PDF 文档构建的 Docker 镜像：
```

File: resources/tidb-pdf-generation-tutorial.md [hunk 12]
```diff
@@ -0,0 +1,118 @@
...
+
+## 环境准备
+
+以下准备工作只需要在第一次生成 PDF 时进行，以后再次生成 PDF 时可直接跳过。
+
+### 准备 1：安装并配置 Docker 环境
+
+> 大约耗时：30 分钟
+
+1. 安装 [Docker](https://docs.docker.com/get-docker/)。
```
Comment (Oreoxmt) [inline resources/tidb-pdf-generation-tutorial.md:18]:
```suggestion
1. 安装 [Docker Desktop](https://docs.docker.com/get-docker/)。
```

File: resources/tidb-pdf-generation-tutorial.md [hunk 13]
```diff
@@ -0,0 +1,118 @@
...
+
+本教程提供了一种自助生成 PDF 格式的 TiDB 文档的方法。通过该方法，你可以在本地对 TiDB 文档目录进行自由排序和删减，定制输出符合特定场景需求的 PDF。
+
+## 环境准备
+
+以下准备工作只需要在第一次生成 PDF 时进行，以后再次生成 PDF 时可直接跳过。
+
+### 准备 1：安装并配置 Docker 环境
+
+> 大约耗时：30 分钟
```
Comment (Oreoxmt) [inline resources/tidb-pdf-generation-tutorial.md:16]:
```suggestion
> 预计耗时：30 分钟
```

File: resources/tidb-pdf-generation-tutorial.md [hunk 14]
```diff
@@ -0,0 +1,118 @@
...
+
+4. 在 macOS Terminal 或者 Windows PowerShell 运行以下命令拉取文档构建的 Docker 镜像：
+
+    ```bash
+    docker pull andelf/doc-build:0.1.9
+    ```
+
+### 准备 2：将 TiDB 文档仓库克隆到本地
+
+> 大约耗时：10 分钟
```
Comment (Oreoxmt) [inline resources/tidb-pdf-generation-tutorial.md:37]:
```suggestion
> 预计耗时：10 分钟
```

File: resources/tidb-pdf-generation-tutorial.md [hunk 15]
```diff
@@ -0,0 +1,118 @@
...
+
+TiDB 中文文档仓库的地址为 <https://github.com/pingcap/docs-cn>，英文文档仓库的地址为 <https://github.com/pingcap/docs>。
+
+下面的步骤以克隆 TiDB 中文文档为例：
+
+1. 打开 TiDB 中文文档仓库地址：<https://github.com/pingcap/docs-cn>
```
Comment (Oreoxmt) [inline resources/tidb-pdf-generation-tutorial.md:43]:
```suggestion
1. 打开 TiDB 中文文档仓库地址：<https://github.com/pingcap/docs-cn>。
```

File: resources/tidb-pdf-generation-tutorial.md [hunk 16]
```diff
@@ -0,0 +1,118 @@
...
+
+3. 通过以下任一方法将 TiDB 中文文档仓库克隆到本地。
+
+    - 方法一：使用 GitHub Desktop 客户端。
+
+        1. 安装并打开 [GitHub Desktop](https://desktop.github.com/)。
+        2. 在 GitHub Desktop 中，点击 **File** > **Clone Repository**。
+        3. 在 **GitHub.com** 选项卡中的 **Your Repositories** 下，选择你 Fork 出来的文档仓库，点击右下角的 **Clone**。
+
+    - 方法二：使用以下 Git 命令行。
```
Comment (Oreoxmt) [inline resources/tidb-pdf-generation-tutorial.md:55]:
```suggestion
    - 方法二：使用以下 `git` 命令。
```

File: resources/tidb-pdf-generation-tutorial.md [hunk 17]
```diff
@@ -0,0 +1,118 @@
...
+
+        ```
```
Comment (Oreoxmt) [inline resources/tidb-pdf-generation-tutorial.md:57]:
```suggestion
        ```shell
```

File: resources/tidb-pdf-generation-tutorial.md [hunk 18]
```diff
@@ -0,0 +1,118 @@
...
+        git clone git@github.com:$user/docs-cn.git # 将 `$user` 替换为你的 GitHub ID
+
+        cd $working_dir/docs-cn
+        git remote add upstream git@github.com:pingcap/docs-cn.git # 添加上游仓库
+        git remote -v
+        ```
+
+## 操作步骤
+
+> 大约耗时：操作只需要 2 分钟，PDF 生成需要等待 0.5 到 1 个小时
```
Comment (Oreoxmt) [inline resources/tidb-pdf-generation-tutorial.md:68]:
```suggestion
> 预计耗时：操作只需要 2 分钟，PDF 生成需要等待 0.5 到 1 个小时
```

File: resources/tidb-pdf-generation-tutorial.md [hunk 19]
```diff
@@ -0,0 +1,118 @@
...
+
+1. 确保你本地 TiDB 文档仓库中的文件为上游 GitHub 文档仓库中的最新版本。
+
+2. 按照你的文档需求，对 TiDB 文档目录进行自由排序和删减。
```
Comment (Oreoxmt) [inline resources/tidb-pdf-generation-tutorial.md:72]:
```suggestion
2. 按照你的需求，对 TiDB 文档目录进行自由排序和删减。
```

File: resources/tidb-pdf-generation-tutorial.md [hunk 20]
```diff
@@ -0,0 +1,118 @@
...
+
+    1. 打开位于本地文档仓库根目录的 `TOC.md` 文件。
+    2. 编辑 `TOC.md` 文件。例如，你可以删除所有不需要的文档章节标题和链接。
+
+3. 按照 `TOC.md` 文件将所有文档中的章节整合到一个 Markdown 文件中。
+
+    1. 打开桌面的 Docker 应用程序。
```
Comment (Oreoxmt) [inline resources/tidb-pdf-generation-tutorial.md:79]:
```suggestion
    1. 启动 Docker 应用程序。
```

File: resources/tidb-pdf-generation-tutorial.md [hunk 21]
```diff
@@ -0,0 +1,118 @@
...
+    2. 在 macOS Terminal 或者 Windows PowerShell 运行以下命令，进入文档构建的 Docker 镜像：
```
Comment (Oreoxmt) [inline resources/tidb-pdf-generation-tutorial.md:80]:
```suggestion
    2. 在 macOS Terminal 或者 Windows PowerShell 运行以下命令，进入 PDF 文档构建的 Docker 镜像：
```

File: resources/tidb-pdf-generation-tutorial.md [hunk 22]
```diff
@@ -0,0 +1,118 @@
...
+
+4. 生成文档 PDF：
+
+    ```bash
+    bash scripts/generate_pdf.sh
+    ```
+
+    **期望输出：**
+
+    PDF 生成所需要的时间与文档的大小有关。对于完整的 TiDB 中文用户文档，大约需要 1 个小时左右。生成完成后，你将在文档所在的文件夹看到新生成的 PDF 文件 `output.pdf`。
```
Comment (Oreoxmt) [inline resources/tidb-pdf-generation-tutorial.md:118]:
```suggestion
    PDF 生成所需要的时间与文档的大小有关。对于完整的 TiDB 中文用户文档，大约需要 1 个小时。生成完成后，你将在文档所在的文件夹看到新生成的 PDF 文件 `output.pdf`。
```

File: resources/tidb-pdf-generation-tutorial.md [hunk 23]
```diff
@@ -0,0 +1,118 @@
...
+本教程提供了一种自助生成 PDF 格式的 TiDB 文档的方法。通过该方法，你可以在本地对 TiDB 文档目录进行自由排序和删减，定制输出符合特定场景需求的 PDF。
+
+## 环境准备
+
+以下准备工作只需要在第一次生成 PDF 时进行，以后再次生成 PDF 时可直接跳过。
+
+### 准备 1：安装并配置 Docker 环境
+
+> 大约耗时：30 分钟
+
```
Comment (Oreoxmt) [inline resources/tidb-pdf-generation-tutorial.md:17]:
```suggestion

下面以 macOS 或 Windows 安装 Docker Desktop 为例。

```

**Review by Oreoxmt — APPROVED** [review]
Rest LGTM

**Comment by shawn0915** [comment]
remove-translation doing

**Pull request merged**